docs/serverless/08-03-bind-service-instance.md

```diff
@@ -153,8 +153,8 @@ Follows these steps:
 
     </details>
     <details>
-    <summary label="console-ui">
-    Console UI
+    <summary label="ui">
+    UI
```

Comment:
```suggestion
    Console UI
```
Refer to [this comment](https://github.com/kyma-project/kyma/pull/9296#discussion_r478304363).